docs/docs_source/wopi/file_contents/PutFile.rst

```diff
@@ -12,10 +12,18 @@ PutFile
 
     The PutFile operation updates a file's binary contents.
 
-    Office Online will always pass the lock ID in the **X-WOPI-Lock** request header. If the file is currently
-    associated with a lock established by the :ref:`Lock` operation or the :ref:`UnlockAndRelock` operation, the
-    host must ensure that a "lock mismatch" response (:http:statuscode:`409`) is returned if the lock passed does not
-    match the lock currently on the file, or if the file has been locked by someone other than Office Online.
+    ..  include:: /fragments/priorlock.rst
+
+    ..  include:: /fragments/lock409.rst
+
+    ..  include:: /fragments/no_lock_id.rst
+
+    During :ref:`document creation<Create New>`, Office Online will make a PutFile request without a prior
+    :ref:`Lock` request. In this case, Office Online will set the **X-WOPI-Size** header to 0. Thus, when a host
```

Comment:
I don't think we set `X-WOPI-Size` anywhere as part of PutFile. Isn't `X-WOPI-Size` something host returns as part of `GetFile`?